### PR TITLE
media-video/mpv: verbump to 0.20.0, adjust default USE flags

### PR DESCRIPTION
**Pull request by Coacher**

### CODE REVIEW

**Comment by Coacher** [comment]
@wraeth, once this PR is merged you can safely remove the last mpv tarball from your devspace.

**Comment by Coacher** [comment]
CC @gentoo/proxy-maint

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: media-video/mpv/mpv-0.20.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+		opengl? ( x11-libs/libXdamage )
+		vdpau? ( >=x11-libs/libvdpau-0.2 )
+		xinerama? ( x11-libs/libXinerama )
+		xscreensaver? ( x11-libs/libXScrnSaver )
+		xv? ( x11-libs/libXv )
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	${PYTHON_DEPS}
+	>=dev-lang/perl-5.8
```
Comment (monsieurp) [inline media-video/mpv/mpv-0.20.0.ebuild]: This version of perl doesn't exist in the tree anymore. :)

File: media-video/mpv/mpv-0.20.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+pkg_preinst() {
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+
+	local softvol_0_18_1=0
+	for rv in ${REPLACING_VERSIONS}; do
```
Comment (monsieurp) [inline media-video/mpv/mpv-0.20.0.ebuild]: could you also make `rv` local pls?

File: media-video/mpv/mpv-0.20.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+}
+
+pkg_postrm() {
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+}
+
+src_test() {
+	cd "${S}"/build/test || die
+	for test in *; do
```
Comment (monsieurp) [inline media-video/mpv/mpv-0.20.0.ebuild]: Ditto here with `test`.

File: media-video/mpv/mpv-0.20.0.ebuild (hunk 1 above)
Comment (Coacher) [inline media-video/mpv/mpv-0.20.0.ebuild]: Fixed.

File: media-video/mpv/mpv-0.20.0.ebuild (hunk 2 above)
Comment (Coacher) [inline media-video/mpv/mpv-0.20.0.ebuild]: Sure. Fixed.

File: media-video/mpv/mpv-0.20.0.ebuild (hunk 3 above)
Comment (Coacher) [inline media-video/mpv/mpv-0.20.0.ebuild]: Fixed.

**Comment by Coacher** [comment]
Thank you, @monsieurp.

**Pull request closed without merging**